ansible/roles/github-bot/templates/environment-file.j2

```diff
@@ -11,4 +11,4 @@ JENKINS_JOB_CITGM={{ envs.jenkins_job_citgm }}
 JENKINS_BUILD_TOKEN_CITGM={{ envs.jenkins_build_token_citgm }}
 JENKINS_JOB_NODE={{ envs.jenkins_job_node }}
 JENKINS_BUILD_TOKEN_NODE={{ envs.jenkins_build_token_node }}
-JENKINS_WORKER_IPS=147.75.70.237,147.75.73.189,169.60.150.88,51.15.200.62,51.15.218.201,147.75.74.174,147.75.193.230,163.172.186.154
+JENKINS_WORKER_IPS=147.75.70.237,147.75.73.189,169.60.150.88,51.15.200.62,51.15.218.201,147.75.74.174,147.75.193.230,163.172.186.154,147.75.66.203,147.75.81.67
```

Comment:
Does the bot need to be redeployed to pick up the new IP addresses?

Reply:
did it manually